### DISCUSSION TITLE
overview/

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#188] [Announcements]
# overview/

第一章：小册简介 PS：为了增加这份小册的趣味性，我特意为此追加了两个虚拟角色，一个二哥，一个三妹，二哥负责教，三妹负责学。这样大家在学习 Java 的时候代入感会更强烈一些，希望这样的设定能博得大家的欢心。

https://tobebetterjavaer.com/overview/

**Comment by CharlesShan-hub** [comment]
三妹好漂亮哈哈哈
